modules/mod_wrapper/src/Helper/WrapperHelper.php

```diff
@@ -32,7 +32,7 @@ class WrapperHelper
      *
      * @since   1.5
      */
-    public static function getParams(&$params)
+    public function getParams(&$params)
```

Comment:
Well done, but there is still work to be done.

Give the non static function a new name, eg:
```suggestion
    public function getParamsWrappper(Registry $params, SiteApplication $app)
```
Indicate the static function as deprecated:
`public function getParams(&$params)`

Examples can be found in the already converted modules.

Reply:
Resolved...thanks for hints. I just do not know, why build failed now